core/src/main/java/org/apache/brooklyn/util/core/xstream/ClassRenamingMapper.java

```diff
@@ -42,7 +42,7 @@ public ClassRenamingMapper(Mapper wrapped, Map<String, String> nameToType) {
     
     @Override
     public Class<?> realClass(String elementName) {
-        Optional<String> elementNameOpt = Reflections.tryFindMappedName(nameToType, elementName);
+        Maybe<String> elementNameOpt = Reflections.findMappedNameMaybe(nameToType, elementName);
```

Comment:
Why rename this? It was previously following the same naming conventions as guava's `Iterables` (e.g. `tryFind` versus `find`).

Also, if changing it to a `Maybe` then should we explicitly check for it returning `Maybe.of(null)`, rather than risking a NPE later?